package.json

```diff
@@ -64,11 +65,15 @@
     "commander": "^2.9.0",
     "css-loader": "^0.28.0",
     "cssnano": "^3.10.0",
+    "enzyme": "^2.8.0",
     "express": "^4.14.1",
     "file-loader": "^0.11.1",
     "friendly-errors-webpack-plugin": "^1.6.1",
     "history": "^3.2.1",
+    "identity-obj-proxy": "^3.0.0",
     "isomorphic-style-loader": "^1.1.0",
+    "jest": "^19.0.2",
```

Comment:
@christophehurpeau, correct me if I'm mistaken, but the issue says explicitely that
> [CLI dependencies] will be stored inside {A}/node_modules/B/node_modules/.bin/*
If we use `npmRun`, the issue is already taking care of, as we are going to search in `vitaminjs/node_modules`, and if not found resolve to the next parent directory.